README.md

```diff
@@ -173,11 +173,15 @@ grunt watch
 Run the unit tests with a local server that supports PHP. Ensure that you run the site from the root directory, not the "test" directory. No database is required. Pre-configured php local servers are available for Windows and Mac. Here are some options:
 
 - Windows: [WAMP download](http://www.wampserver.com/en/)
-- Mac: [MAMP download](http://www.mamp.info/en/index.html)
 - Linux: [Setting up LAMP](https://www.linux.com/learn/tutorials/288158-easy-lamp-server-installation)
 - [Mongoose (most platforms)](http://code.google.com/p/mongoose/)
+- Mac: [MAMP download](http://www.mamp.info/en/index.html)
```

Comment:
They still need to be configured, even on a Mac and it's not that easy.

The best solution would be to fix #1755 but it needs some effort.